FEM/pcs_dm.cpp

```diff
@@ -772,19 +772,21 @@ double CRFProcessDeformation::Execute(int loop_process_number)
 
 				Error = Norm / InitialNorm;
 				ErrorU = NormU / InitialNormU0;
-				if (Norm < Tolerance_global_Newton && Error > Norm)
```

Comment:
I suggest making a separate PR changing and discussing the condition for convergence. This PR  should focus only on the material models.